moved/src/move_execution/tests.rs

```diff
@@ -1475,6 +1573,10 @@ impl ModuleCompileJob {
                 "evm_admin".into(),
                 NumericalAddress::parse_str("0x1").unwrap(),
             ),
+            (
```

Comment:
We can create a convenience function just like `add_custom_framework_paths` to include these named addresses.

Reply:
Done in https://github.com/MovedNetwork/op-move/pull/153/commits/d1d5cb6e09fa9a5f5fba22fe36170d0953b93d62